# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1304725095 25200
#      Fri May 06 16:38:15 2011 -0700
# Branch stable
# Node ID 20e64dab6896b4559f437217f916eeab3d3906eb
# Parent  5fd90b7cb78761f85afdb8b91db1f0b1fc663f8a
doc: Correct spacing in erf and gamma documentation in INFO mode.

* mappers.cc (erf, erfinv, erfc, erfcx, gamma): Improve docstring.

diff --git a/src/mappers.cc b/src/mappers.cc
--- a/src/mappers.cc
+++ b/src/mappers.cc
@@ -545,21 +545,22 @@ Compute the error function,\n\
 $$\n\
  {\\rm erf} (z) = {2 \\over \\sqrt{\\pi}}\\int_0^z e^{-t^2} dt\n\
 $$\n\
 @end tex\n\
 @ifnottex\n\
 \n\
 @example\n\
 @group\n\
-                         z\n\
-                        /\n\
+@c spacing appears odd here, but is correct after Makeinfo\n\
+                          z\n\
+                         /\n\
 erf (z) = (2/sqrt (pi)) | e^(-t^2) dt\n\
-                        /\n\
-                     t=0\n\
+                         /\n\
+                      t=0\n\
 @end group\n\
 @end example\n\
 \n\
 @end ifnottex\n\
 @seealso{erfc, erfcx, erfinv}\n\
 @end deftypefn")
 {
   octave_value retval;
@@ -602,17 +603,17 @@ erf (z) = (2/sqrt (pi)) | e^(-t^2) dt\n\
 */
 
 DEFUN (erfinv, args, ,
     "-*- texinfo -*-\n\
 @deftypefn {Mapping Function} {} erfinv (@var{x})\n\
 Compute the inverse error function, i.e., @var{y} such that\n\
 \n\
 @example\n\
-  erf(@var{y}) == @var{x}\n\
+  erf (@var{y}) == @var{x}\n\
 @end example\n\
 @seealso{erf, erfc, erfcx}\n\
 @end deftypefn")
 {
   octave_value retval;
   if (args.length () == 1)
     retval = args(0).erfinv ();
   else
@@ -639,17 +640,17 @@ Compute the inverse error function, i.e.
 DEFUN (erfc, args, ,
     "-*- texinfo -*-\n\
 @deftypefn {Mapping Function} {} erfc (@var{z})\n\
 Compute the complementary error function,\n\
 @tex\n\
 $1 - {\\rm erf} (z)$.\n\
 @end tex\n\
 @ifnottex\n\
-@code{1 - erf (@var{z})}.\n\
+@w{@code{1 - erf (@var{z})}}.\n\
 @end ifnottex\n\
 @seealso{erfcx, erf, erfinv}\n\
 @end deftypefn")
 {
   octave_value retval;
   if (args.length () == 1)
     retval = args(0).erfc ();
   else
@@ -669,17 +670,17 @@ Compute the complementary error function
 DEFUN (erfcx, args, ,
     "-*- texinfo -*-\n\
 @deftypefn {Mapping Function} {} erfcx (@var{z})\n\
 Compute the scaled complementary error function,\n\
 @tex\n\
 $z^2 (1 - {\\rm erf} (z))$.\n\
 @end tex\n\
 @ifnottex\n\
-@code{z^2*(1 - erf (@var{z}))}.\n\
+@w{@code{z^2*(1 - erf (@var{z}))}}.\n\
 @end ifnottex\n\
 @seealso{erfc, erf, erfinv}\n\
 @end deftypefn")
 {
   octave_value retval;
   if (args.length () == 1)
     retval = args(0).erfcx ();
   else
@@ -874,17 +875,18 @@ Compute the Gamma function,\n\
 $$\n\
  \\Gamma (z) = \\int_0^\\infty t^{z-1} e^{-t} dt.\n\
 $$\n\
 @end tex\n\
 @ifnottex\n\
 \n\
 @example\n\
 @group\n\
-             infinity\n\
+@c spacing appears odd here, but is correct after Makeinfo\n\
+              infinity\n\
              /\n\
 gamma (z) = | t^(z-1) exp (-t) dt.\n\
              /\n\
           t=0\n\
 @end group\n\
 @end example\n\
 \n\
 @end ifnottex\n\
